doc/dev_guide.rst

```diff
@@ -167,10 +167,13 @@ you do.
 HyperSpy uses the `pytest <http://doc.pytest.org/>`_ library for testing. The
 tests reside in the ``hyperspy.tests`` module. 
 
-First ensure pytest and its plugins are installed:
+First ensure pytest and its plugins are installed by:
 
 .. code:: bash
-
+   
+   # Either
+   pip install hyperspy[test]
```

Comment:
Not sure if this would install the release version, but most likely, the developer want to install with the pip development mode:
```
pip install .[test]
```
Need to be checked if this work nicely.

Reply:
I think you mean pip install -e .[test]?

Reply:
I can confirm that `pip install -e .[test]` installs hyperspy as a dev install and also installs the dependencies.